charts/flagsmith/Chart.yaml

```diff
@@ -2,13 +2,18 @@ apiVersion: v2
 name: flagsmith
 description: Flagsmith
 type: application
-version: 0.2.22
-appVersion: "2.9"
+version: 0.3.0
+appVersion: 2.9.1
```

Comment:
Is there a reason to lock this to a specific version rather than e.g. `2.9`? If we specified `2.9` we would receive bug fixes etc?

Reply:
Pining version is the best practice. Especially when you use it as a container image tag. Let me explain it as an example. We have deployed Flagsmith to our Kubernetes cluster but some node dies. It means Kubernetes deletes the original pod and schedule a new one on the new node. The new version has been released. So new node actually fetches `2.9.1` instead of `2.9.0`. That was what happened to us. We have had deployed `2.9` (`2.9.0`) with has worked but new node pulls `2.9.1` with has non-working change (https://github.com/Flagsmith/flagsmith-charts/pull/49).

PS: there can be another discussion if change this big (non-backward compatible) should be released as a "patch" version.